packages/happner-client/test/example.com.cert

```diff
@@ -1,17 +1,17 @@
 -----BEGIN CERTIFICATE-----
-MIICpDCCAYwCCQDc/3wrjZJF+DANBgkqhkiG9w0BAQsFADAUMRIwEAYDVQQDDAls
```

Comment:
figure out why these get checked in every time, they may need to be git deleted and ignored

Reply:
See change to happn-3 transport service above. This should no longer be recreated on each test run

Reply:
See changes to happn-3 transport service...